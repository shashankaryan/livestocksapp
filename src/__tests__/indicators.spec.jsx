import React from 'react'
import { shallow } from 'enzyme' // render, static is also available. but they are slower.
import Indicator from '../Indicators'

test('Table Item renders correctly', () => {
	const component = shallow(<Indicator />)
	expect(component).toMatchSnapshot()
})
