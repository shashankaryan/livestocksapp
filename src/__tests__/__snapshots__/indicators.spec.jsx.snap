// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Item renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    style={
        Object {
            "marginLeft": "10%",
          }
    }
>
    <styled.h3>
        INDICATORS: 
    </styled.h3>
    <styled.p>
        Price Increased
    </styled.p>
    <styled.p>
        Price Decreased
    </styled.p>
</div>,
  "nodes": Array [
    <div
      style={
            Object {
                  "marginLeft": "10%",
                }
      }
>
      <styled.h3>
            INDICATORS: 
      </styled.h3>
      <styled.p>
            Price Increased
      </styled.p>
      <styled.p>
            Price Decreased
      </styled.p>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Indicator />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          style={
                    Object {
                              "marginLeft": "10%",
                            }
          }
>
          <styled.h3>
                    INDICATORS: 
          </styled.h3>
          <styled.p>
                    Price Increased
          </styled.p>
          <styled.p>
                    Price Decreased
          </styled.p>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          style={
                    Object {
                              "marginLeft": "10%",
                            }
          }
>
          <styled.h3>
                    INDICATORS: 
          </styled.h3>
          <styled.p>
                    Price Increased
          </styled.p>
          <styled.p>
                    Price Decreased
          </styled.p>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Indicator />,
}
`;
